hebiros_gazebo_plugin/include/hebiros_gazebo_controller.cpp

```diff
@@ -178,6 +184,105 @@ void HebirosGazeboController::SetDefaultGains(std::shared_ptr<HebirosGazeboGroup
     hebiros_group->settings.effort_gains.ki.push_back(0);
     hebiros_group->settings.effort_gains.kd.push_back(0.001);
   }
+  if (model_name == "X8_3" && control_strategy == 2) {
```

Comment:
No enum values for the `control_strategy` ?

Reply:
I'd be happy to change that to an enum; there's a lot of the code that really needs to be restructured, but the changes start spidering pretty quickly.